std_msgs/Header header
map_sense/RawGPUPlanarRegion[] regions
uint16 numOfRegions